diff --git a/node_modules/react-native-sqlite-storage/platforms/android/build.gradle b/node_modules/react-native-sqlite-storage/platforms/android/build.gradle
index ff79b10..e681a94 100644
--- a/node_modules/react-native-sqlite-storage/platforms/android/build.gradle
+++ b/node_modules/react-native-sqlite-storage/platforms/android/build.gradle
@@ -1,8 +1,8 @@
 buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
+   repositories {
+    google()
+    mavenCentral()
+}
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
